events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+        }, ONE_SECOND_MS);
+    }
+
+    function bubbleOff() {
+        Entities.callEntityServerMethod(bubble, "bubbleOff");
+        Messages.sendMessage(TRIVIA_CHANNEL, JSON.stringify({ type: "game off" }));
+        if (intervalBoard) {
+            Script.clearInterval(intervalBoard);
+        }
+        type = null;
+        category = null;
+        difficulty = null;
+    }
+  
+    function findTargets() {
+        Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element) {
+            var name = Entities.getEntityProperties(element, ['name']).name;
+            if (name.indexOf("Trivia") !== -1) {
+                if (name.indexOf("Light") !== -1) {
+                    lights.push(element);
+                } else if (name.indexOf("Correct") !== -1) {
+                    correctHighlights.push(element);
+                } else {
+                    switch (name) {
+                        case "Trivia Question":
+                            questionText = element;
+                            break;
+                        case "Trivia Answer":
+                            answerText = element;
+                            break;
+                        case "Trivia Choice Text Green":
+                            choiceTexts[0] = element;
+                            break;
+                        case "Trivia Choice Text Yellow":
+                            choiceTexts[1] = element;
+                            break;
+                        case "Trivia Choice Text Red":
+                            choiceTexts[2] = element;
+                            break;
+                        case "Trivia Choice Text Blue":
+                            choiceTexts[3] = element;
+                            break;
+                        case "Trivia Timer Countdown":
+                            timer = element;
+                            break;
+                        case "Trivia Avatar Counter Total":
+                            avatarCounter = element;
+                            break;
+                        case "Trivia Player Game Zone":
+                            gameZone = element;
+                            gameZoneProperties = Entities.getEntityProperties(gameZone, 
+                                ["id", "position", "dimensions", "rotation", "userData"]);
+                            break;
+                        case "Trivia Prize Amount":
+                            prizeDisplay = element;
+                            break;                            
+                        case "Trivia Bubble":
+                            bubble = element;
+                            break;
+                    }
+                }
+            }
+        });
+    }
+
+    function onClicked() { 
+        if (open) {
+            tablet.gotoHomeScreen();
+        } else {
+            tablet.gotoWebScreen(appPage);
+        }
+    }
+
+    // function getQuestion() {
```

Comment:
any reason this is still around?

Reply:
yes, it is for when a custom database is desired like was used for halloween or the giraffage event.

Reply:
that sounds like something to put in the README rather than keeping dead code around! please move this to the readme with some description around it

Reply:
Done.